tfhe/benches/shortint/glwe_packing_compression.rs

```diff
@@ -0,0 +1,68 @@
+use criterion::{black_box, criterion_group, Criterion};
+use rayon::iter::{IntoParallelIterator, ParallelIterator};
+use tfhe::shortint::glwe_compression::COMP_PARAMS_FOR_MESSAGE_2_CARRY_2_KS_PBS;
+use tfhe::shortint::prelude::*;
+
+fn glwe_packing(c: &mut Criterion) {
+    #[allow(clippy::single_element_loop)]
+    for (name, param, comp_param, number_to_pack) in [(
+        "1",
```

Comment:
what's the name "1" ?

Reply:
It was when running the benches the names chosen for the first set of parameters
Will remove the loop and the name

Reply:
Removed loop